LandSandBoat/server
# Sacrifice fix

**_I affirm:_**

## What does this pull request do?

The WHM spell Sacrifice had no logic in it to support its interaction with Afflatus Solace for transferring multiple (also Erasable) status effects at once. This adds it in.

## Steps to test these changes

Apply several status effects to a character in a party (either naturally or with !addeffect)
Have a White Mage use the job ability Afflatus Solace then cast Sacrifice on the target
Up to 7 status effects (the base effects that can be transferred without Afflatus Solace + those that can be Erased) should be transferred from the target to the caster
